### DISCUSSION TITLE
learning/ulpb

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#24] [General]
# learning/ulpb

双拼 在 Asuka Minato 的安利下，种下了契机的种子。然后在一个契机到来的时候开始学习并使用双拼了。确实非常舒服。 对比 优势： 按键次数少，打字快， 打出的都是精确拼音，减少了检索的时间。（参考劣势 2.） 没啥学习成本，我练了 90min 基本上记住了，之后练习也方便，毕竟重度电脑使用者每天都需要打字。 一次学习，终身受益。 劣势： 双手...

https://absx.pages.dev/learning/ulpb.html

**Comment by s-chance** [comment]
双拼应该也有词库，然后词库通过[多设备同步 | oh-my-rime 输入法](https://www.mintimate.cc/zh/guide/deviceSync.html)，感觉也能弥补不足？或者可以用oh-my-rime的全拼+双拼混合模式？

**Reply by lxl66566** [reply]
对的，rime 双拼是直接用全拼词库。
词库同步我[使用 git 定期同步](https://github.com/lxl66566/rime)，仅限 *.dict.yaml。userdb 我直接放弃了，通过手动补充 dict，尽可能做到不依赖 userdb。

**Comment by mcthesw** [comment]
有试过辅助码方案吗？感觉双拼的重码率还是有一些的，靠辅助码可以一定程度上缓解这个问题：
`vi ug ji` -> `zhi sheng ji` -> 直升机？只升级？
`vi ug ji u` -> `zhi sheng ji + 第一个字具有一个“十”` -> 直升机
`vi ug ji o` -> `zhi sheng ji + 第一个字具有一个“口”` -> 只升级

**Reply by lxl66566** [reply]
我目前没有用辅助码或形码，确实有重码率高这个问题，自制词库可以缓解一些...
这个辅助码看起来确实挺不错的，收益高，不过复杂度也高，实际使用还得看看有没有 rime 方案实现

**Reply by mcthesw** [reply]
现在我用的是这个：[moran](https://github.com/rimeinn/rime-moran)
辅助码现在比较让我头疼的是辅助位置也有方案差异：
- 例如上述例子的辅助码是给第一个字补充形码
- 也有方案是给最后一个字补充了一个形码
- 或者干脆把每个字都变成双拼+形码的三键形式，感觉就有点得不偿失了
